app/controllers/concerns/hyrax/embargoes_controller_behavior.rb

```diff
@@ -0,0 +1,43 @@
+module Hyrax
+  module EmbargoesControllerBehavior
+    extend ActiveSupport::Concern
+    include Hyrax::ManagesEmbargoes
+    include Hyrax::Collections::AcceptsBatches
+
+    included do
+      # include the render_check_all view helper method
+      helper ::BatchEditsHelper
```

Comment:
Can you say why this needed to be added? I'm not seeing `render_check_all` called anywhere in this PR.